graph/mutators/asset.js

```diff
@@ -56,12 +56,25 @@ const closeNow = async (ctx, id) =>
     }
   );
 
+/**
+ * scrapeAsset will force scrape an asset.
+ *
+ * @param {Object} ctx graphql context
+ * @param {String} id the asset's id to scrape
+ */
+const scrapeAsset = async (ctx, id) => {
+  const { services: { Scraper } } = ctx;
```

Comment:
I think there need to be a connectors in there, like:
  const Scraper = ctx.connectors.services.Scraper;

Reply:
Yuup! We have a fix already for that #1425